<?xml version="1.0"?>

<launch>
  <!-- Navigation Master -->
  <node pkg="navigation_master" type="navigation_master_v1_3.py" name="navigation_master" output="screen">
<!--    <param name="directory" value="/home/atlas80evo/catkin_ws/src/atlas80evo/traj"/> -->
    <param name="directory" value="/home/atlas80evo/catkin_ws/src/atlas80evo/traj/json_path"/>
    <param name="battery_threshold" value="50"/> <!-- [%] -->
    <param name="timer" value="10"/> <!-- [sec] -->
  </node>

  <!-- Doing Table Pickup Action -->
<!--  <node pkg="navigation_master" type="action_pickup_table.py" name="action_pickup_table" output="screen"/> -->
  <node pkg="navigation_master" type="action_pickup_table_v2.py" name="action_pickup_table_v2" output="screen"/>
<!--  <node pkg="under_table" type="table_finder.py" name="table_finder"/> -->
  <node pkg="under_table" type="table_finder_v2.py" name="table_finder_v2" output="screen"/>

  <!-- Doing Table Dropoff Action --> 
  <node pkg="navigation_master" type="action_dropoff_table_v2.py" name="action_dropoff_table_v2" output="screen"/>
  <node pkg="turning_180" type="turning_180_v4.py" name="turning_180_v4"/>
  <node pkg="under_table" type="leave_table.py" name="leave_table"/>

</launch>
